environment.yml

```diff
@@ -10,7 +11,7 @@ dependencies:
   - pip
   - pycocotools
   - pyproj>=2.2
-  - python>=3.6
+  - python>=3.8
```

Comment:
Can you explain this change?

Reply:
Need to revert. Was modifying this to create different conda envs for testing.

Reply:
For testing I would recommend doing everything in pure pip. Conda takes way too long. With pip you can just pip install a single package at a different version.